jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -162,6 +163,16 @@ async def application_delete(
     """
     Delete application from the database and S3 given it's id.
```

Comment:
quick fix in the docstring while we're here :)
```suggestion
    Delete application from the database and S3 given its id.
```